pennylane/transforms/decompose.py

```diff
@@ -214,6 +294,44 @@ def eval(self, jaxpr: "jax.core.Jaxpr", consts: Sequence, *args) -> list:
             for const, constvar in zip(consts, jaxpr.constvars, strict=True):
                 self._env_map[constvar] = const
 
+            if qml.decomposition.enabled_graph():
+
+                operations = []
+
+                # Evaluate the jaxpr of the dynamic decomposition
+                # and create a decomposition graph from the known operations.
+                for eq in jaxpr.eqns:
+
+                    # Create the graph creation if eq
+                    # - hasn't been registered yet
+                    # - is a primitive operator
+                    # - has a DropVar output
+                    # - has no Var inputs (skip parametric ops)
```

Comment:
We don't want to skip all parametric ops. Parametric ops are totally supported unless their resource params are abstract. So most parametric ops are perfectly fine. The only problematic op that I can think of is the `Pow` with an abstract `z`.